src/components/projecteditor/compiler.js

```diff
@@ -164,7 +164,13 @@ export default class Compiler extends Component {
                     }
                     else {
                         (data.errors || []).map((row)=>{
-                            this.consoleRows.push({channel:3,msg:row.formattedMessage});
+                            if(row.severity==="warning")
```

Comment:
Please follow same this code conventions for now: 

```js

if (row.severity === "warning") {
...
} else {
...
}
```

Also the same as the this.consoleRows.push code (I would use the same format as the one in the else clause)

Reply:
Also, do we have any other level apart from "warning" and "error"  which would be interesting to distinguish? This way whatever is not a warning will be perceived as an error.

Reply:
@javier-tarazaga No. At the moment, `severity` is set to either `warning` or `error`.
Reference: https://github.com/ethereum/solidity/blob/develop/libsolidity/interface/StandardCompiler.cpp#L50

You could differentiate between the different error types (`Type` enum, mentioned in previous comment) by reading the `type` field, but I don't see the point in this case.